mqtt/mqtt-bridge/src/messages.rs

```diff
@@ -58,19 +58,25 @@ impl TryFrom<TopicRule> for TopicMapper {
 }
 
 /// Handle events from client and saves them with the forward topic
-pub struct StoreMqttEventHandler<S> {
+pub struct StoreMqttEventHandler<S>
+where
+    S: StreamWakeableState + Send + Sync,
+{
     topic_mappers: HashMap<String, TopicMapper>,
     topic_mappers_updates: TopicMapperUpdates,
-    store: PublicationStore<S>,
+    store: Arc<PublicationStore<S>>,
```

Comment:
This Arc can likely be avoided by making PublicationStore cloneable, it should also already be Send and Sync as the member of PublicationStore is PublicationInner which is an Arc<Mutex>